jenkins-integration/beaker/install/foss/40_clone_test_catalogs.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Clone test-catalogs repo onto master'
+
+# This is awful - we clone locally and then SCP the whole thing over.
+# See QENG-2556 for adding a proper way to do this in Beaker.
+tmp_repo = "/tmp/test-catalogs-#{Process.pid}"
```

Comment:
there are beaker util functions for asking it for a temp dir... but those may only work for the SUT?

failing that you still might want to do something like `mktemp -d` rather than hard-coding a location in `/tmp`.